code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -46,7 +47,8 @@
 	var/cache_key = "[base_icon]-[material.name]"
 	if(isnull(stool_cache[cache_key]))
 		var/image/I = image('icons/obj/furniture.dmi', base_icon)
-		I.color = material.icon_colour
+		if(applies_material_colour) //VOREStation Add - Goes with added var

```

Comment:
The Downstream doesn't like when we leave their comments in for things we steal.